semantic-conventions/src/tests/data/yaml/errors/stability/experimental_attr_stable_member.yaml

```diff
@@ -0,0 +1,13 @@
+groups:
+  - id: test
+    type: attribute_group
+    brief: 'test'
+    attributes:
+      - id: enum_attr
```

Comment:
Nit: Even though it's the default, I think it's nice to explicitly add the stability attribute here, so it's very clear what the test/file is about.

Reply:
Given the problems we've had with experimental things being treated as "defacto stable", I think I'd support requiring `stability` attribute on everything

Reply:
fixed the test!

Agree that requiring stability to be set explicitly is good - reflected it in this issue https://github.com/open-telemetry/semantic-conventions/issues/763

Reply:
updated this PR to require stability on enum members in the last commit.